internal/pkg/cli/pipeline_deploy.go

```diff
@@ -173,6 +181,22 @@ func (o *deployPipelineOpts) Execute() error {
 	return nil
 }
 
+func (o *deployPipelineOpts) validatePipelineName() error {
+	// TODO: After pipeline file structure changes, validate with o.ws.ListPipelines() instead.
```

Comment:
nit: can we remove this TODO? you'll be addressing this soon anyway so I'm not super worried. 

side note: This is probably just me but I've never seen an effective TODO comment 🙈 I feel like they just pollute the codebase with things that will never happen and create confusion if they're not addressed quickly